drivers/storage/portworx/component/portworx_api.go

```diff
@@ -202,19 +202,31 @@ func (c *portworxAPI) createDaemonSet(
 
 	//  If px version is greater than 2.13 then update daemonset when csi-node-driver-registrar image changes if CSI is enabled
 	pxVersion := pxutil.GetPortworxVersion(cluster)
+	removeCSIRegistrar := false
+	addCSIRegistrar := false
```

Comment:
Can we have single boolen variable like updateCSIRegistrar rather than having two?